src/game/N3UIDebug.cpp

```diff
@@ -166,6 +168,74 @@ void CN3UIDebug::RenderGameMetrics() {
     ImGui::End();
 }
 
+void CN3UIDebug::RenderGameTimeAndWeather() {
+    ImGui::Begin(IMGUI_WND_ID_TIMEANDWEATHER);
```

Comment:
Maybe we don't need another window? For 3 controls perhaps another window will be an overkill. See example:
![image](https://user-images.githubusercontent.com/90905675/200449277-e2ae761e-fee2-43b6-b6fc-066abc6c9dd1.png)

If the window gets too full, we can also 1) rename it to something more generic, 2) use ImGui's `ImGui::CollapsingHeader`.

Reply:
Changed in this commit: https://github.com/ko4life-net/ko/pull/170/commits/1c1b662d8b7ae298a63841382256bae2391f7854

and also implemented collapsing headers in this commit: https://github.com/ko4life-net/ko/pull/170/commits/dd6056c035c24b9bc29e7e8365db8043c5f2d358